models/migrations/v96.go

```diff
@@ -36,10 +36,9 @@ func deleteOrphanedAttachments(x *xorm.Engine) error {
 					return err
 				}
 
-				_, err := sess.ID(attachment.ID).NoAutoCondition().Delete(attachment)
```

Comment:
this is also a unrelated change
(deleting the db entry after succesfull file-deletion ... sure but this is not KanBan related :/ )